src/use-calendars.ts

```diff
@@ -10,3 +13,25 @@ export const useCalendars: DPUseCalendars = ({ selectedDates, state }) => {
     weekDays: createWeekdays(calendars[0], state.config),
   };
 };
+
+export const useCalendarActions: DPUseCalendarActions = ({
```

Comment:
This is more date actions or config actions, better names would be `useDateActions` or `useConfigActions`